t/t3903-stash.sh

```diff
@@ -632,6 +632,47 @@ test_expect_success 'stash create - no changes' '
 	test_must_be_empty actual
 '
 
+test_expect_success 'stash create with --include-untracked' '
+    git init repo &&
+    cd repo &&
+    echo committed >file1 &&
+    git add file1 &&
+    git commit -m "initial commit" &&
```

Comment:
It would be more concise to run `test_commit file1`...

Also, wouldn't it be possible to reuse [what had already been set up by an earlier test case](https://github.com/gitgitgadget/git/blob/d5a62cb2af298beaa88e4c7fac0f6d69e1ae8e87/t/t3903-stash.sh#L61-L63)?